### ISSUE TITLE
Add a context example of CO2 emission

### ISSUE THREAD

**Issue by Akronix** [mlco2/impact#7]
Raw numbers are meaningless if there is nothing to compare with.
I have no idea if 2 kg of CO2 is at all a big quantity, medium or small. I guess the situation is similar to most of the humans.

I suggest to provide some rudimentary examples of carbon emission: 100 km run by a car, hours of your refrigerator, years of life of a human breathing O2 and expiring CO2, among other ideas. I guess which one to choose would depend on the magnitude of carbon to be displayed and compared with)

**Comment by vict0rsch** [comment]
You are right. I'm sorry I missed your comment but it is indeed something we should include!

**Comment by IamLRBA** [comment]
Hello @Akronix and @vict0rsch , i was wondering if this issue has been attended to already or if it is out of order right now. I thought it would be a good issue to kickstart my contributions to this project.
